federation/source-config-handlers/src/main/java/org/codice/ddf/admin/sources/wfs/persist/CreateWfsSourcePersistMethod.java

```diff
@@ -85,6 +88,9 @@ public CreateWfsSourcePersistMethod(ConfigurationHandler handler,
     @Override
     public Report persist(WfsSourceConfiguration configuration) {
         Configurator configurator = configuratorFactory.getConfigurator();
+        configurator.startFeature(WFS_CORE_FEATURE);
```

Comment:
You shouldn't need to start this, the WFS features should be starting these features up already.